nucypher/cli/commands/deploy.py

```diff
@@ -207,7 +208,7 @@ def create_actor(self,
                 password = get_client_password(checksum_address=deployer_address)
 
         # Produce Actor
-        testnet = deployer_interface.client.chain_name != NetworksInventory.MAINNET
+        testnet = deployer_interface.client.chain_name != PUBLIC_CHAINS[1]  # Mainnet
```

Comment:
:+1: - Yes I think this will do the trick.

Reply:
Retracing your thinking here - this makes the `PUBLIC_CHAINS` list case sensitive, right?

Reply:
Instead of caring about case sensitivity, now you are comparing apples to apples - the chain_name returns a value from the PUBLIC_CHAINS dict, so we just compare it to the mainnet value in the same dict.

Reply:
Ah, right :100:

Reply:
🍎2🍎